lib/Twig/Cache/Filesystem.php

```diff
@@ -46,7 +46,7 @@ public function generateKey($name, $className)
      */
     public function load($key)
     {
-        @include_once $key;
+        if (file_exists($key)) include_once $key;
```

Comment:
The time lapsus between the check and the include is in the nano seconds range.

If you are able to reproduce that condition even once, tell me, I will stop using my computer.